DFW.h

```diff
@@ -1,18 +1,33 @@
-// Serial controller library for DFWireless controllers
+/**
+ * Serial controller library for DFWireless controllers

```

Comment:
We might have scope creep with the DFW library.  Now instead of just handling the controller, it is also handling all of the competition stuff.